### PR TITLE
Add variables to adjust heading colors

### PR DESCRIPTION
**Pull request by deiteris**
Add `--arc-font-headline-color` and `--arc-font-title-color` to allow modification of heading colors.

### CODE REVIEW

File: src/ApiSecurityDocumentation.js [hunk 1]
```diff
@@ -46,12 +46,14 @@ export class ApiSecurityDocumentation extends AmfHelperMixin(LitElement) {
         font-weight: var(--arc-font-headline-font-weight);
         letter-spacing: var(--arc-font-headline-letter-spacing);
         line-height: var(--arc-font-headline-line-height);
+        color: color: var(--arc-font-headline-color);
```
Comment (jarrodek) [inline src/ApiSecurityDocumentation.js]: Double color definition. This won't fly ;)

**Review by jarrodek — CHANGES_REQUESTED** [review]
Please, fix the double `color` and we are ready to merge.

File: src/ApiSecurityDocumentation.js (hunk 1 above)
Comment (deiteris) [reply src/ApiSecurityDocumentation.js]: How embarrassing... Thanks for the notice, will fix it :)

**Review by jarrodek — APPROVED** [review]
LGTM

**Pull request merged**